Cython/Utility/AsyncGen.c

```diff
@@ -351,90 +348,6 @@ static PyType_Spec __pyx_AsyncGenType_spec = {
     Py_TPFLAGS_DEFAULT | Py_TPFLAGS_HAVE_GC | Py_TPFLAGS_HAVE_FINALIZE, /*tp_flags*/
```

Comment:
I  wonder if I should add `Py_TPFLAGS_IMMUTABLETYPE` to all of these? That's probably the main visible difference between the heap-type and static-type version, and think they genuinely should be immutable.

Reply:
Try it. Sounds right.